### DISCUSSION TITLE
Protected Memo

### DISCUSSION THREAD

**Discussion by blaine07** [usememos/memos#535] [Q&A] [answered]
Can someone explain what Protected memo does? How is it different than Private?

**Comment by Zeng1998** [comment, marked as answer]
visible only to the logged in users

**Comment by blaine07** [comment]
Is that not what private is?Thanks,Blaine--Sent by carrier pigeonOn Nov 22, 2022, at 08:38, Zeng1998 ***@***.***> wrote:﻿
visible only to the logged in users

**Comment by boojack** [comment]
@blaine07 PTAL https://github.com/usememos/memos/issues/126#issuecomment-1190077245
